src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,61 @@
+import { ReactPlugin, withAITracking } from '@microsoft/applicationinsights-react-js';
+import { ApplicationAnalytics, ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ComponentType } from 'react';
+import ITelemetry from './ITelemetry';
+
+class Telemetry implements ITelemetry {
+  private appInsights: ApplicationInsights;
+  private config: any;
+
+  constructor() {
+    const areWeInDev = process.env.NODE_ENV === 'development';
+
+    this.config = {
+      instrumentationKey: process.env.REACT_APP_INSTRUMENTATION_KEY,
+      disableExceptionTracking: true,
```

Comment:
Do you mind explain why is `disableExceptionTracking` set to true here? Are you disabling exception auto collection?

Reply:
Yes, I am disabling exception auto collection.